### ISSUE TITLE
Sound Effect When Removing or Adding Pages

### ISSUE THREAD

**Issue by Fatih20** [xournalpp/xournalpp#1856]
Pretty self-explanatory. Here's a video with the supposed sound. https://www.youtube.com/watch?v=JOpedahoKoc
For adding a page, you can have "Paper 4" sound, for removing a page, you can have "Paper Crumpled 01" sound.
It'd be nice polish to this software, in my opinion.

**Comment by Technius** [comment]
I don't think this adds any value in terms of usability, but I am open to arguments for it.
